packages/atomic/src/images/main-clear.svg

```diff
@@ -1,6 +1,5 @@
 <svg focusable="false" enable-background="new 0 0 13 13" viewBox="0 0 13 13"
-    xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Clear">
-    <title>Clear</title>
+    xmlns="http://www.w3.org/2000/svg" role="img">
```

Comment:
Why remove the title? There was a good reason to add one back in the JSUI i believe

Reply:
Oops, my bad. I removed this when trying some things out. Re-adding it. 👍 

And, the reason was a request from Intel in september.